packages/generator/src/types.ts

```diff
@@ -1,11 +1,3 @@
-import type { Schema, PDFRenderProps } from '@pdfme/common';
```

Comment:
We just use plugin.

This was a welcome side effect.
Thank you for making this proposal.